### PR TITLE
feat(cli): Read and reinitialize worker certificate authority

### PR DESCRIPTION
**Pull request by irenarindos**
Addition of CLI functions for the worker certificate authority read and reinitialize methods.

### CODE REVIEW

File: internal/api/genapi/input.go [hunk 1]
```diff
@@ -782,6 +782,10 @@ var inputStructs = []*structInfo{
 		fieldFilter:         []string{"private_key"},
 		recursiveListing:    true,
 	},
+	{
+		inProto: &workers.Certificate{},
+		outFile: "workers/certificate.gen.go",
+	},
```
Comment (jefferai) [inline internal/api/genapi/input.go:788]:
```suggestion
	},
		{
		inProto: &workers.CertificateAuthority{},
		outFile: "workers/certificate_authority.gen.go",
	},
```

File: internal/cmd/commands/workerscmd/certificate-authority.go [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+			"",
+			"  Reinitialize the certificate authority used to authorize Boundary workers:",
+			"",
+			`    $ boundary workers certificate-authority reinitialize`,
+			"",
+			"",
+		}) + c.Flags().Help()
+	}
+	return base.WrapForHelpText([]string{
+		"Usage: boundary workers certificate-authority [options]",
```
Comment (jefferai) [inline internal/cmd/commands/workerscmd/certificate-authority.go:68]: See internal/cmd/common/help.go  line 44: the general pattern here is to say something high level, then have an example command. It's a subtle difference but one that makes it clear that the user didn't run the command the wrong way just missing `read`, that `read` is simply _one_ example of something that can be done. Then the "Please see the {{hyphentype}}s subcommand..." bit makes it clear to look at the available subcommands.

**Review by jefferai — APPROVED** [review]
Looks really good! Just one wording suggestion.

File: internal/cmd/commands/workerscmd/certificate-authority.go (hunk 2 above)
Comment (irenarindos) [reply internal/cmd/commands/workerscmd/certificate-authority.go:67]: Thanks, I've updated this!

**Pull request merged**